Information/CenterOfMass.FCMacro

```diff
@@ -1,891 +1,966 @@
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+# coding: utf-8
+#
+# Compute and show the center of mass for multiple solids
 #
 # Usage:
-# 1. select one or multiple solids
-# 2. launch the macro.
+# 1. Select one or multiple solids.
+# 2. Launch the macro.
 # 3. You'll have a window listing the solids. You can put the density of your
-#    material or choose from a large list of predefined materials. (If you know
-#    your densities in another unit system, I used FreeCAD convert system to
-#    propose different units.)
+#    material in different unit systems or choose from predefined materials.
 #
 # Options:
-# * You can also display where is the center of mass.
-# * An import button allows you to quickly load saved densities (even if it's
-#   not a .csv file from the macro, as soon as there is a column named "Density"
-#   it will be loaded.)
+# * Color the solids according to density.
+# * Display where is the center of mass.
+# * Import densities (even if it's not a .csv file from the macro, as soon as
+#   there is a column named "Density").
+# * Save densities in the document or export to a csv-file
+#
+# Credits:
+# 2018 - present: schupin
+# 2022 - present: SyProLei project (Saarland University)
 #
-# Attribution:
-# Icons import.png, compute.png, save.png, material.png, and com.png are
-# from https://www.freepik.com/
-
-from __future__ import division
-from __future__ import unicode_literals
 
 __Name__ = 'CenterOfMass'
-__Comment__ = 'Computation of the center of mass for multiple solids'
-__Author__ = 'schupin'
-__Version__ = '0.4.3'
-__Date__ = '2022-02-19'
+__Comment__ = 'Compute and show the center of mass for multiple solids'
+__Author__ = 'chupins, s-quirin'
+__Version__ = '0.5'
+__Date__ = '2022-03-29'
 __License__ = 'LGPL-3.0-or-later'
 __Web__ = 'https://forum.freecadweb.org/viewtopic.php?f=24&t=31883'
 __Wiki__ = 'https://www.freecadweb.org/wiki/Macro_CenterOfMass'
 __Icon__ = 'https://www.freecadweb.org/wiki/images/d/d7/Centomass.png'
 __Help__ = 'Select one or more bodies and launch'
-__Status__ = 'alpha'
-__Requires__ = 'freecad 0.18'
-__Communication__ = 'https://forum.freecadweb.org/viewtopic.php?p=265270#p265270'
-__Files__ = 'CenterOfMass/Macro_CenterOfMass_import.png,CenterOfMass/Macro_CenterOfMass_compute.png,CenterOfMass/Macro_CenterOfMass_save.png,CenterOfMass/Macro_CenterOfMass_material.png,CenterOfMass/Macro_CenterOfMass_com.png,CenterOfMass/Macro_CenterOfMass_colorify.png'
+__Status__ = 'Alpha'
+__Requires__ = 'FreeCAD >= 0.19'
+__Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=24&t=31883'
+__Files__ = ''
 
-#todo :
-#   -dockable
-#   -check rotation of parts
+# Todo:
+# - resetting stylesheet of QDoubleSpinBox
+# - error with draft array of meshes (relevant?)
+# Ideas:
+# - moments of inertia with arrows that allow the user to see the relative magnitudes
 
 import csv
-import glob
 import os
 import math
 
 from PySide import QtCore, QtGui
```

Comment:
Comment FreeCAD's stealing the PySide module:
`from PySide import QtCore, QtGui  # FreeCAD's PySide!`